src/Scores.cpp

```diff
@@ -116,6 +121,8 @@ void ScoreHolder::printPSM(ostream& os, bool printDecoys, bool printExpMass) {
   }
 }
 
+
```

Comment:
Remove this space.

Reply:
Fixed: https://github.com/percolator/percolator/commit/e56ae7dcaaa192c379958cf14dc9465b8c11b6c5